test/clojure-mode-util-test.el

```diff
@@ -46,7 +46,7 @@
                (bb-edn-src (expand-file-name "src" temp-dir)))
           (write-region "{}" nil bb-edn)
           (make-directory bb-edn-src)
-          (expect  (clojure-project-dir bb-edn-src)
+          (expect  (expand-file-name (clojure-project-dir bb-edn-src))
```

Comment:
Not sure why this is needed. This test was failing only on windows.
I don't have a windows machine to reproduce/fix it properly.